### ISSUE TITLE
look-at error using a-frame 0.5 and targeting by id

### ISSUE THREAD

**Issue by darioodiaz** [ngokevin/kframe#72]
I have the following code:

```
<body style='margin : 0px; overflow: hidden;'>
    <a-scene physics="debug: true">
        <a-assets>
            <a-asset-item src="/models/mario/mario-sculpture.obj" id="mario-model"></a-asset-item>
            <a-asset-item src="/models/mario/mario-sculpture.mtl" id="mario-mtl"></a-asset-item>
        </a-assets>
        <a-camera id="player" look-controls wasd-controls="acceleration: 200">
            <a-cursor></a-cursor>
        </a-camera>
        <a-plane static-body material="color: green;" position="0 -3 0" width="30" height="30" rotation="-90 0 0"></a-plane>
        <a-entity look-at="#player" dynamic-body="shape: sphere; sphereRadius: 2.8; mass: 1" scale="0.05 0.05 0.05" position="0 0 -5" obj-model="obj: #mario-model; mtl: #mario-mtl"></a-entity>
    </a-scene>
</body>
```

 but I have that error when the look-at component loads

`Uncaught (in promise) TypeError: isCoordinates is not a function
at Object.parse (http://127.0.0.1:8080/scripts/aframe-look-at.min.js:70:12)
    at i.parseAttrValueForCache (http://127.0.0.1:8080/scripts/aframe.min.js:244:2297)
    at i.updateCachedAttrValue (http://127.0.0.1:8080/scripts/aframe.min.js:244:2120)
    at i.updateProperties (http://127.0.0.1:8080/scripts/aframe.min.js:244:2627)
    at i.module.exports.Component (http://127.0.0.1:8080/scripts/aframe.min.js:244:1485)
    at new i (http://127.0.0.1:8080/scripts/aframe.min.js:244:4852)
    at HTMLElement.value (http://127.0.0.1:8080/scripts/aframe.min.js:236:4592)
    at HTMLElement.value (http://127.0.0.1:8080/scripts/aframe.min.js:236:5965)
    at e (http://127.0.0.1:8080/scripts/aframe.min.js:236:5464)
    at Array.forEach (native)
`

Any idea?

**Comment by darioodiaz** [comment]
debugging a little bit, the A-frame function util is isCoordinate and the code has isCoordinates

**Comment by ngokevin** [comment]
This has been fixed on the master branch of K-Frame. Have not released until 0.6.0 is out

**Comment by darioodiaz** [comment]
Great ! thanks :)
